src/components/User/KycRequests/KycRequest.List.vue

```diff
@@ -166,6 +162,7 @@ export default {
         address: '',
         type: ''
       },
+      specificUserAddress: '',
```

Comment:
I’d like the way with `filters.requestor` you got in `AssetRequestList.vue`, let’s omit that usage of a proxy-variable cuz it complicates things a lot